src/redux/tripsRedux.js

```diff
@@ -14,6 +14,21 @@ export const getFilteredTrips = ({trips, filters}) => {
   // TODO - filter by duration
 
   // TODO - filter by tags
+  if (filters.tags.length) {
+    const filtered = [];
+    output.forEach(trip => {
+      let matches = true;
+      filters.tags.forEach(tag => {
+        if (!trip.tags.includes(tag)) {
+          matches = false;
+        }
+      });
+      if (matches) {
+        filtered.push(trip);
+      }
+    });
+    return filtered;
+  }
```

Comment:
ja bym to skrócił - ten kod był bardziej żeby pokazać "jak zrobić to łopatologicznie"